opm/material/fluidmatrixinteractions/EclMaterialLawManager.hpp

```diff
@@ -741,6 +741,45 @@ class EclMaterialLawManager
             else if (!slgofTables.empty())
                 readGasOilEffectiveParametersSlgof_(effParams, Swco, tolcrit,
                                                     slgofTables.getTable<SlgofTable>(satRegionIdx));
+            else if ( !tableManager.getSgofletTable().empty() ) {
+                const auto& letSgofTab = tableManager.getSgofletTable()[satRegionIdx];
+                const std::vector<Scalar> dum;
+
+                effParams.setApproach(SatCurveMultiplexerApproach::LETApproach);
+                auto& realParams = effParams.template getRealParams<SatCurveMultiplexerApproach::LETApproach>();
+
+                // S=(So-Sogcr)/(1-Sogcr-Sgcr-Swco),  krog = Krt*S^L/[S^L+E*(1.0-S)^T]
+                const Scalar s_min_w = letSgofTab.s2_critical;
+                const Scalar s_max_w = 1.0-letSgofTab.s1_critical-Swco;
+                const std::vector<Scalar>& letCoeffsOil = {s_min_w, s_max_w,
+                                                           static_cast<Scalar>(letSgofTab.l2_relperm),
+                                                           static_cast<Scalar>(letSgofTab.e2_relperm),
+                                                           static_cast<Scalar>(letSgofTab.t2_relperm),
+                                                           static_cast<Scalar>(letSgofTab.krt2_relperm)};
+                realParams.setKrwSamples(letCoeffsOil, dum);
```

Comment:
I think this approach works, but I cannot say I find it easy to understand. I am here talking about copying the data to a dumb vector, passing it through an interface that has no obvious connection to what is really going on, and unpacking on the inside.

Unless my brain is fooling me, at this point the type of `realParams` should be `ThoPhaseLETCurvesParam`, and then you could make one or a few interface methods for it that are reasonably named and has properly named parameters etc.

Reply:
The idea was to adapt to the interface already used by the `PiecewiseLinearTwoPhaseMaterial`. To my understanding, introducing tailored methods for the `TwoPhaseLETCurvesParams` would have to impact `PiecewiseLinearTwoPhaseMaterialParams` as well due to the multiplexer construction.

Reply:
Supplied a comment to `dum` hinting at what is going on.